.github/workflows/release-published.yml

```diff
@@ -195,3 +195,31 @@ jobs:
         unzip '*liquibase-maven-plugin*.jar' -d liquibase-maven-plugin/
         rm -rf *.jar *.zip
         aws s3 sync . s3://javadocsliquibasecom-origin --only-show-errors
+
+  publish_to_github_packages:
+    name: Publish artifacts to Github Packages
+    runs-on: ubuntu-20.04
+    needs: [ setup ]
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Java for publishing to GitHub Repository
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+          cache: 'maven'
+          server-id: github
+
+      - name: Version artifact
+        run: mvn versions:set -DnewVersion="${{ needs.setup.outputs.version }}"
+
+      # Publish to GitHub Packages
+      - name: Publish package to Github
+        run: mvn -B clean deploy -DskipTests=true
```

Comment:
Is Maven configured to only deploy the modules we care about to GitHub packages? I am probably missing that somewhere.

Reply:
Yeah we have `<maven.deploy.skip>false</maven.deploy.skip>` set in the pom files of all submodules (except core), but in liquibase-extension-testing it is set to `true` in the profile `github-package` when maven detects the env var `TARGET_ARTIFACT_REPOSITORY` is set to `github`. This deploy step is separate from the one that actually pushes to sonatype (which does not set the env var), so in this new step we'll be pushing the core artifact and extension-testing artifact on release only to github packages.